packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import contentStyles from "./ContentStyles.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.string.isRequired,
+  description: PropTypes.node,
+  isOpen: PropTypes.bool.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  onClose: PropTypes.func,
+  onConfirm: PropTypes.func.isRequired,
+  heading: PropTypes.string.isRequired,
```

Comment:
same goes for this one, you should be able to pass whatever you want here.

Reply:
Since this component should always have an `<h>` heading, by making this a node, you give the consumer more flexibility, but you put more responsibility on them.  So I'm not sure sure about always reaching for `PropTypes.node`.   In fact, I'd consider taking on the additional complexity of supporting both `string` and `node`, and if it is a plain string, automatically including it inside a `<Heading level={2} displayLevel={5} />` (or however we want to code/display this heading to be properly accessible and consistent with the design system).

Reply:
In fact... since `description` is a `node`, if this prop was simply not required, then the user could put whatever markup they want there, and opt out of this "shortcut" prop for providing a heading.